### DISCUSSION TITLE
Temperature in Parameter Sweep Simulation

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#791] [Q&A] [answered]
Is it possible to simulate temperature dependence in a circuit using Parameter Sweep?
If it is, an example would be greatly appreciated.

I have tried setting "Temp" in a transistor model to a symbol MYTEMP and vary it in Parameter Sweep, no success.

BN

**Comment by ra3xdh** [comment, marked as answer]
The `TEMPER` represents a global temperature in Ngspice. But the DC sweep requires `TEMP` as the parameter to sweep. These variable names are a bit ambiguous. Here is an example of the temperature sweep.

![image](https://github.com/ra3xdh/qucs_s/assets/4920080/a7be5ae2-862a-450a-a3c4-68fdfe0f6243)

**Comment by bnilsson11** [comment]
Thanks, that solves it for me.
